chrysom/basicClient.go

```diff
@@ -93,31 +92,6 @@ const (
 // Items is a slice of model.Item(s) .
 type Items []model.Item
 
-// GetLogger returns a logger from the given context.
-type GetLogger func(context.Context) *zap.Logger
-
-// SetLogger embeds the `Listener.logger` in outgoing request contexts for `Listener.Update` calls.
-type SetLogger func(context.Context, *zap.Logger) context.Context
```

Comment:
GetLogger & SetLogger types are only used to simplify fx dependency injection (why they were moved in fx.go)